WebRoot/js/zimbraMail/calendar/view/ZmCalDayView.js

```diff
@@ -319,6 +319,13 @@ function(abook) {
     // Fix for bug: 66603. Attaching a scroll function.
     document.getElementById(this._tabsContainerDivId).onscroll = func;
 
+	if (AjxEnv.isModernIE) {
```

Comment:
not sure if this issue is with IE11 and above only, as this check will only match IE11 and edge versions
and will not handle IE 10 and below versions

Reply:
although not very important to handle as microsoft itself has stopped supporting older IE versions

Reply:
It matches IE11 only.
For the latest MS Edge based on chromium, it is regarded as Chrome.
```
AjxEnv.isChrome: true
AjxEnv.isIE: false
AjxEnv.isMSEdge: false
AjxEnv.isModernIE: false
```
When using MS Edge based on chromium + IE mode, it is regarded as IE11. It is the same as native IE11.
```
AjxEnv.isChrome: false
AjxEnv.isIE: false
AjxEnv.isMSEdge: false
AjxEnv.isModernIE: true
```
For IE10 or below versions, Zimbra does not support them. In addition, we cannot install IE10 on windows 10 (downgrade install is not allowed.) I didn't write the condition as `AjxEnv.isIE || AjxEnv.isModernIE` because we cannot test it on them and they are out of support of Zimbra.
Theoretically, `AjxEnv.isModernIE` is false on them and then the issue is reproducible on them.